spec/features/create_environment_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'support/test_data_builder'
+
+describe "Creating an environment" do
+  let(:path) { "/environments/test" }
+  let(:headers) { {'CONTENT_TYPE' => 'application/json'} }
+  let(:response_body) { JSON.parse(last_response.body, symbolize_names: true)}
+  let(:environment_hash) do
+    {
+      label: "Test",
+      owners: [ { name: "Team Awesome", contacts: ["awesome@company.org"] } ]
```

Comment:
Can't think of a better word than "contacts" but I want it to be an array strings that people can put free form text in that could be email addresses/Slack URLs, @ names etc.

For Pactflow, we would link the environment to one or more teams, rather than use the owners field.

Reply:
Yeah that makes sense to me. Another option would be to allow each item in `contacts` to be a free-form object, so that it have additional context to it.

Or each `Contact` could have a shape of:
```
{
   "contact": string,
   "details:" {},
}
```

Simple (string) might be better as a starting point.